src/webchat-ui/components/WebchatUI.tsx

```diff
@@ -368,6 +409,7 @@ export class WebchatUI extends React.PureComponent<React.HTMLProps<HTMLDivElemen
 							className="webchat-root"
 							aria-labelledby="webchatHeaderTitle"
 							role="region"
+							onKeyDown={enableFocusTrap && open ? this.handleKeydown : () => {}}
```

Comment:
I'd move the `open` check to `handleKeyDown` and always trigger it.

Reply:
Fixed